bin/resources/GameIndex.yaml

```diff
@@ -18880,7 +18922,7 @@ SLES-54624:
   name: "Samurai Warriors 2 - Empires"
   region: "PAL-E"
 SLES-54626:
-  name: "An American Tail"
+  name: "American Tail, An"
```

Comment:
no need to change this

Reply:
Every "The" are dropped to the end of any title for alpha sorting. Why not for "An" (although it doesn't change greatly the sorting)?

Reply:
ah my bad , my mistake then